packages/hoppscotch-backend/src/admin/admin.service.ts

```diff
@@ -240,14 +240,26 @@ export class AdminService {
       teamID,
       user.value.uid,
     )();
-    if (E.left(isUserAlreadyMember)) {
+    if (E.isLeft(isUserAlreadyMember)) {
```

Comment:
```suggestion
    if (E.isLeft(teamMember)) {
```
As it is not a boolean type value, we can change the variable name.

Reply:
okay